app/pages/system/networking/IpPoolsPage.tsx

```diff
@@ -112,6 +112,7 @@ export default function IpPoolsPage() {
   const { table, query } = useQueryTable({
     query: ipPoolList(),
     columns,
+    rowHeight: 'large',
```

Comment:
The irony is there are no IPv6 ranges, so we don't strictly need to accommodate them in the rows. I'm going to take a quick look at making this sneakier while making sure it behaves reasonably if IPv6 shows up without us noticing.